config/rendered/dev/cspr/westus3.yaml

```diff
@@ -342,7 +343,7 @@ svc:
       kvName: ah-cspr-se-usw3-1
       kvSoftDelete: false
     infraAgentPool:
-      azCount: 1
+      azCount: 3
```

Comment:
Wasn't quite sure if this should actually be the number of available zones or just `1` always?